internal/upgrader/upgrader.go

```diff
@@ -131,17 +153,6 @@ func performUpgrade(api CFClient, upgradableInstances []ccapi.ServiceInstance, p
 	return nil
 }
 
-func performCheckUpToDate(upgradableInstances []ccapi.ServiceInstance, log Logger) error {
-	err := performDryRun(upgradableInstances, log)
```

Comment:
question: I am a bit unsure this is the best strategy. By reusing the dry run we make sure that whatever we say is out of date, would be also detected by the dry run and also be upgraded by the plugin if we just run it without any flags. Here we are in a situation when a bug could be introduced/fixed in one path and not the other.